apps/antalmanac/src/actions/AppStoreActions.ts

```diff
@@ -122,15 +122,16 @@ export const loadSchedule = async (userId: string, rememberMe: boolean) => {
 
                 if (scheduleSaveState === undefined) {
                     openSnackbar('error', `Couldn't find schedules for username "${userId}".`);
-                } else if (await AppStore.loadSchedule(scheduleSaveState)) {
-                    openSnackbar('success', `Schedule for username "${userId}" loaded.`);
```

Comment:
This check fails if no courses are in the schedule. This results in `queryWebsoc` never being called, and never erroring out. Skeleton mode would not end up being triggered.